docs/mkdocs.yml

```diff
@@ -215,6 +215,9 @@ nav:
         - run get: 'cmd/flow/run/run-get.md'
         - run list: 'cmd/flow/run/run-list.md'
         - run resubmit: 'cmd/flow/run/run-resubmit.md'
+    - Power Platform (pp):
+      - managementapp:
+        - managementapp add: 'cmd/pp/managementapp/app-add.md'
```

Comment:
The file name doesn't match the command name. We have a command name `pp managementapp add`, but the file name + path would indicate `pp managementapp app add`.